subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1303,11 +1359,17 @@ static inline void ticker_job_worker_bh(struct ticker_instance *instance,
 				instance->ticks_slot_previous = 0U;
 			}
 
-			/* save current ticks_slot_previous */
-			if (ticker->ticks_slot != 0U) {
-				instance->ticker_id_slot_previous = id_expired;
-				instance->ticks_slot_previous =
-					ticker->ticks_slot;
+			/* If a reschedule is set pending, we will need to keep
+			 * the slot_previous information
+			 */
+			if (!TICKER_RESCHEDULE_PENDING(ticker)) {
+				/* save current ticks_slot_previous */
+				if (ticker->ticks_slot != 0U) {
+					instance->ticker_id_slot_previous =
+						id_expired;
+					instance->ticks_slot_previous =
+						ticker->ticks_slot;
+				}
```

Comment:
This change to make sure that when rescheduling a node, the id_slot_previous and slot_previous information is not cleared, as we need it for the next ticker_worker.